RangeOperators.md

```diff
@@ -0,0 +1,80 @@
+# Range Operators
+
+## Convention
+
+Prefer the half-open range operator `(a..<b)` over the closed range operator `(a...b)` when not including the final value.
+
+## Rationale
+
+Much like using `<` instead of `<=`, using the half-open range operator allows you to exclude the final value without the need of additional arithmetic. That being said, if you intend to use the final, the closed range operator is still suitable.
+
+## Examples
+
+### Array Iteration
+```swift
+let oneToFive = ["one", "two", "three", "four", "five"]
+let count = oneToFive.count
+
+// really bad: off by one
+for i in 0...count {
+    print(oneToFive[i])
+}
+//one
+//two
+//three
+//four
+//five
+//Fatal error: Index is out of range
+
+// bad:
```

Comment:
Can we explain why this is bad? Something like "unnecessary arithmetic" calling back to one of the phrases you introduced in the Rationale?

Reply:
Same comment elsewhere